modules/openapi-generator-online/src/main/java/org/openapitools/codegen/online/service/Generator.java

```diff
@@ -181,7 +182,7 @@ private static String generate(String language, GeneratorInput opts, Type type)
 
     private static File getTmpFolder() {
         try {
-            File outputFolder = File.createTempFile("codegen-", "-tmp");
+            File outputFolder = Files.createTempFile("codegen-", "-tmp").toFile();
             outputFolder.delete();
             outputFolder.mkdir();
```

Comment:
Vulnerability still exists here. This does not fix the issue.
This entire API call should be changed to the following:

```suggestion
            File outputFolder = Files.createTempDirectory("codegen-tmp").toFile();
```